src/five/formlib/tests/formlib.txt

```diff
@@ -45,21 +45,30 @@ Let's 'manually' create a Content instance and add it to the folder:
   >>> obj = content.Content('content_1', 'Title', [])
   >>> folder._setObject('content_1', obj)
   'content_1'
-  >>> print folder.content_1.title
+  >>> print (folder.content_1.title)
```

Comment:
```suggestion
  >>> print(folder.content_1.title)
```

As `print` is a function in Python 3 and the parenthesis are a no-op in Python 2, I'd prefer to omit the space between the function and its arguments, so it looks more like Python 3.

There are other code lines where this space can be removed, too.